landing-strip/core/src/main/java/com/novoda/landingstrip/State.java

```diff
@@ -3,6 +3,7 @@
 import android.support.v4.view.ViewPager;
 
 class State {
+    private final IndicatorCoordinatesCalculator indicatorCoordinatesCalculator;
```

Comment:
odd that a class called State has the logic to _calculate_ the indicator position.